fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Alias.java

```diff
@@ -116,14 +115,13 @@ public String toString() {
         return child().toString() + " AS `" + name + "`#" + exprId;
     }
 
-    public <R, C> R accept(ExpressionVisitor<R, C> visitor, C context) {
-        return visitor.visitAlias(this, context);
-    }
-
     @Override
     public Expression withChildren(List<Expression> children) {
         Preconditions.checkArgument(children.size() == 1);
         return new Alias(exprId, children.get(0), name);
     }
 
+    public <R, C> R accept(ExpressionVisitor<R, C> visitor, C context) {
```

Comment:
why just move this function's position?

Reply:
Style: we should put `override function` together